static/js/modules/tables.js

```diff
@@ -160,17 +160,17 @@ var dateColumn = formattedColumn(helpers.datetime, {orderSequence: ['desc', 'asc
 var currencyColumn = formattedColumn(helpers.currency, {orderSequence: ['desc', 'asc']});
 var barCurrencyColumn = barColumn(helpers.currency);
 
-var candidateColumn = formattedColumn(function(data) {
-  if (data) {
-    return buildEntityLink(data.name, helpers.buildAppUrl(['candidate', data.candidate_id]), 'candidate');
+var candidateColumn = formattedColumn(function(data, type, row) {
+  if (row) {
+    return buildEntityLink(row.candidate_name, helpers.buildAppUrl(['candidate', row.candidate_id]), 'candidate');
```

Comment:
Line is too long.<br>Identifier 'candidate_name' is not in camel case.<br>Identifier 'candidate_id' is not in camel case.